packages/nimble-components/src/dialog/styles.ts

```diff
@@ -12,7 +12,9 @@ import {
     smallPadding,
     subtitleFont,
     subtitleFontColor,
-    elevation3BoxShadow
+    elevation3BoxShadow,
+    dialogSmallWidth,
+    dialogSmallMaxHeight
```

Comment:
MaxHeight is a new token suffix and should be added to the list: https://github.com/ni/nimble/blob/main/packages/nimble-components/src/theme-provider/design-token-names.ts#L243

Note the comment at the top, it should come before Height in the list.

Reply:
FYI, that will result in you needing to add an entry here as well: https://github.com/ni/nimble/blob/73b899d1d36e3c04219e36b989a7367bd3800a87/packages/nimble-components/src/theme-provider/tests/tokens.stories.ts#L100

Reply:
added